client/client.go

```diff
@@ -368,6 +368,29 @@ func (o *OpenstackClient) GetImage(nameOrID string) (*images.Image, error) {
 	}); err != nil {
 		return nil, fmt.Errorf("failed to get image with name or id %s: %w", nameOrID, err)
 	}
+	// try again but look for community images
```

Comment:
I think there is no good way to search for `public` AND `community' visibility. so we have to duplicate the code .. 

but we could also simply use `images.ImageVisibility("all")` ?! What do you think ?

Reply:
I think that's a great idea. Better still, we could make this a flag in the config/extra-specs `search-all-images` or something. By default limit to public images, and we switch to `all` if the flag is set to true. We must take care because (*I think*) if we mistakingly use credentials for an admin user, `all` can truly include all images.

Reply:
Okay .. I'll change this. 

The visibility thing is really tricky .. you have `pubilc`, `private`, `shared` and `community` .. but the visibility is only important for  **listing** the images not necessarily for **using**. E.g. you can boot from any community image .. and you can also boot from a `shared` image which you didn't accept (yet) or which you explicitly rejected. 
Hence the allowed owners check is very crucial

Reply:
visibility can now be explicitly set in config and/or extra specs